ta/miss/include/ta_sims.h

```diff
@@ -0,0 +1 @@
+../../sims/include/ta_sims.h
```

Comment:
I don't think this file is needed. `#include <ta_sims.h>` should find the proper header.